crates/core/c8y_translator/fuzz/Cargo.toml

```
@@ -4,7 +4,7 @@ name = "c8y_translator-fuzz"
 version = "0.0.0"
 authors = ["Automatically generated"]
 publish = false
-edition = "2018"
+edition = "2021"
 
```

We miss `rust-version`.
```suggestion
rust-version = "1.58"
```